pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -516,10 +516,13 @@ func TestNSMGR_ShouldCorrectlyAddForwardersWithSameNames(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
 
+	const registryExpiryDuration = time.Second
```

Comment:
Can we have a const for expire testing?

Reply:
Yep, added `sandbox.RegistryExpiryDuration`.